test/conformance/enqueue/urEnqueueReadHostPipe.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+#include <uur/fixtures.h>
+
+using urEnqueueReadHostPipeTest = uur::urHostPipeTest;
+
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(urEnqueueReadHostPipeTest);
+
```

Comment:
just out of curiosity, is there something that blocks us from adding the success test?